src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -102,10 +104,21 @@ export const AddressModal: React.FC<Props> = ({
               onError && onError(message)
             }
 
-            const handleSuccess = address => {
+            const handleSuccess = savedAddress => {
+              if (isDefault) {
+                updateUserDefaultAddress(
+                  // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
blocking: this line was added in a bunch of places when we initially turned on strict null checking, but we should not be adding new ones. If this line is needed to prevent a linting error in this call, we should fix the arguments instead.

Reply:
fixed